qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -226,12 +228,33 @@ def __repr__(self) -> str:
             ")"
         )
 
+    @classmethod
+    def set_truncation(cls, val):
+        """Set the max number of characters to display before truncation/
+        Args:
+            val (int): the number of characters.
```

Comment:
```suggestion
    def set_truncation(cls, val: int) -> None:
        """Set the max number of characters to display before truncation/
        Args:
            val: the number of characters.
```
We normally have typehints on the method itself rather than on the params

Reply:
Thanks. I made a mistake because I've been developing a lot of modules recently.